README.md

```diff
@@ -34,6 +34,27 @@ jobs:
 +        args: --config=.project.php_cs --diff --dry-run
 ```
 
+_to support fixing styles of a pull request via `/php-cs-fixer` keyword use:_
+
+note: make sure you use the same php-version for the fixer action as your projects min-php version requirement. otherwise the fixer might introduce syntax errors.
```

Comment:
dont know howto define the php version the action will/should use

Reply:
It seems inside the container the php binaries are available
https://github.com/localheinz/composer-normalize/blob/2e64a6f173977e40b5516fdbe1c951c2fff9563d/.github/workflows/continuous-integration.yml#L13-L26

Not sure how we should configure them though

Reply:
I guess we can use this action to run the checks in the php version we need:

https://github.com/nanasess/setup-php